### PR TITLE
switch to using prod source system with CICD running (docs generated)

### PR DESCRIPTION
**Pull request by mrmauer**
Final touches before publishing provider
- [x] switch to prod domain in client
- [x] all acceptance tests are succeeding
- [x] examples are cleaned up to provide reference on how to use each individual resource
- [x] set up CI/CD w/ live environment variables
- [ ] ~~end-to-end test go live flow with live DNS providers (custom_domain_verification resource)~~ (tracked elsewhere)
- [x] fix any linter violations

### CODE REVIEW

File: docs/resources/fe_integration.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+# generated by https://github.com/hashicorp/terraform-plugin-docs
+page_title: "propelauth_fe_integration Resource - propelauth"
+subcategory: ""
+description: |-
+  Front-end Integration. This is for configuring the front-end integration for one of your poject's environments.
+---
+
+# propelauth_fe_integration (Resource)
+
+Front-end Integration. This is for configuring the front-end integration for one of your poject's environments.
```
Comment (itailevi98) [inline docs/resources/fe_integration.md]: typo here

```suggestion
Front-end Integration. This is for configuring the front-end integration for one of your project's environments.
```

File: docs/resources/roles_and_permissions.md [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+# generated by https://github.com/hashicorp/terraform-plugin-docs
+page_title: "propelauth_roles_and_permissions Resource - propelauth"
+subcategory: ""
+description: |-
+  Roles and Permissions resource. This is for configuring the basic roles and permissionsrmation in PropelAuth.
+---
+
+# propelauth_roles_and_permissions (Resource)
+
+Roles and Permissions resource. This is for configuring the basic roles and permissionsrmation in PropelAuth.
```
Comment (itailevi98) [inline docs/resources/roles_and_permissions.md]: typo here

```suggestion
Roles and Permissions resource. This is for configuring the basic roles and permissions information in PropelAuth.
```

File: internal/propelauth/client.go [hunk 3]
```diff
@@ -5,29 +5,27 @@ import (
 	"encoding/json"
 	"fmt"
 	"net/http"
-	"net/url"
 	"runtime"
 	"time"
 )
 
-// TODO: FOR THE LOVE OF GOD, DON'T FORGET TO CHANGE THIS TO THE REAL URL.
 const BaseURLTemplate string = "https://api.propelauth.com/iac/%s/project/%s"
```
Comment (itailevi98) [inline internal/propelauth/client.go:12]: 🥇

**Review by itailevi98 — COMMENTED** [review]
all lgtm (aside from the typos), i see that there are still some lint conflicts so ill keep this commented before approving

**Review by itailevi98 — DISMISSED** [review]
lgtm!

**Comment by mrmauer** [comment]
Bypass merging. Not meeting all requirements only b/c the CODEOWNERS file needs updating on the main branch.

**Pull request merged**